package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.14.0",
   "description": "Sushi Unshortens Short Hand Inputs (FSH Compiler)",
   "scripts": {
-    "build": "rm -rf dist && tsc && cp -r src/ig/files dist/ig/files",
+    "build": "rm -rf dist && tsc && cp -r src/ig/files dist/ig/files && cp src/import/generated/*.d.ts dist/import/generated/",
```

Comment:
Does this mean we still build the incorrect .d.ts files with `tsc` and then overwrite them with the `cp`?

Reply:
So far, the best I've been able to figure out with tsc is that if I want it to include the generated js files, I have to set `includeJs: true`, which means that type definition files get generated. I'd much prefer for tsc to notice that I already have my own type definition files, but I'm still trying to figure out how to set up `tsconfig.json` to make that happen.

Reply:
Ok, I have no problem with overwriting really, just wanted to make sure I was understanding correctly.